fs/is-path-empty/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@pnpm/is-path-empty",
+  "version": "1.0.0",
```

Start with 0.0.0. During release it will be bumped to 1.0.0 automatically
```suggestion
  "version": "0.0.0",
```